simcon_project/templates/create_assignment.html

```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE html>
+{% extends "researcher_base.html" %}
+<html lang="en">
```

Comment:
also you need content blocks or it will render your stuff wierd surround with render content blocks